include/config/config_game.h

```diff
@@ -9,6 +9,9 @@
 // Disables AA (Improves console performance but causes visible seams between unconnected geometry).
 //#define DISABLE_AA
 
+// Use a folded polynomial instead of a lookup table
+#define DISABLE_LOOKUP_TABLE
```

Comment:
I think this name is really confusing, I would name it like ``DISABLE_SIN_COS_LOOKUP_TABLE`` or something like this

Reply:
done